terraso_backend/apps/core/models/taxonomy_terms.py

```diff
@@ -39,6 +39,9 @@ def __str__(self):
         return "{}.{}".format(self.type, self.slug)
 
     class Meta(SlugModel.Meta):
+        # we want to define a custom unique constraint here, so we override the default
+        # constraint in parent
+        _override_fields = ["slug"]
```

Comment:
> Question: Is _override_fields just overriding the unique fields definition? If that is the case should it be included in the name of this variable?

Yes, it makes sense to include it in the name. I will do that.

> Question: After this line you still have the UniqueConstraint definition for 2 combined fields (type, slug) how is this interacting with _override_fields?

`override_fields` means that the metaclass will not create a constraint for this field, even though a parent class (`SlugMeta`) has a unique field `"slug"`.  There is probably a cleaner way of doing this, but it would involve adding quite a bit more code. I think it makes sense to make an easier solution here and implement a cleaner solution a bit later on if we see the need.